Fss.Core/Types/Border.fs

```diff
@@ -74,20 +74,22 @@ module Border =
                     match this.Width with
                     | Some width ->
                         match width with
-                        | :? Width as width -> $"{(width :> ICssValue).StringifyCss()} "
-                        | :? ILengthPercentage as il -> $"{lengthPercentageString il} "
+                        | :? Width as width -> $"{(width :> ICssValue).StringifyCss()}"
+                        | :? ILengthPercentage as il -> $"{lengthPercentageString il}"
```

Comment:
This line actually fails in tests.
It works when testing for dotnet, but fails with Fable.

Not sure whats up with that.
But it works if we replace that one line with these two:

```
| :? Percent as p -> stringifyICssValue p
| :? Length as l -> stringifyICssValue l
```

Which is basically what the `lengthPercentageString` does. Maybe a bug in Fable compiler?